### DISCUSSION TITLE
How can I remove borders in terminal applications?

### DISCUSSION THREAD

**Discussion by trixey-mixey** [prasanthrangan/hyprdots#1956] [General]
Executing btop or nvim for example, it doesn't fit all the terminal space but leave some space between gaps and it looks quite ugly. Any way to fix that? It's not depend on either the window is fullscreened or not.
![241031_19h33m11s_screenshot](https://github.com/user-attachments/assets/33cd93f9-88ab-4c05-9f3b-602de8d116d4)

**Comment by kRHYME7** [comment]
edit ~/. config/kitty/kitty.conf remove any padding.
That's the only "intended" way to fix it.
